Expensify/App
# don't show current user in searches

### Details
https://github.com/Expensify/App/issues/4325#issuecomment-890056378

### Fixed Issues

$ #4325 

### Tests || QA Steps

1. Login to newDot
2. Search for your email/phone
3. Your own profile will **NOT** appear as an option 



### Tested On

- Can't test on these two

### Screenshots


#### Web

![image](https://user-images.githubusercontent.com/28243942/128629432-72ec7803-bcba-4b02-8fe7-40c60caf65c7.png)

#### Mobile Web

![mWeb](https://user-images.githubusercontent.com/28243942/128629738-95075ab1-90bb-4a9a-a540-8ec7c7207b59.jpeg)

#### Desktop


#### iOS


#### Android

![mobile](https://user-images.githubusercontent.com/28243942/128629742-8c0a4e2c-3f2c-4de7-9fbf-820100fecbde.jpeg)
